node/src/main.rs

```diff
@@ -5,30 +5,28 @@ mod cli;
 mod log;
 
 use self::cli::cli;
-use forest_libp2p::service::NetworkEvent;
-use futures::prelude::*;
-use network::service::NetworkService;
+use async_std::task;
+use forest_libp2p::service::Libp2pService;
 use slog::info;
-use tokio::runtime::Runtime;
-use tokio::sync::mpsc;
+use std::error::Error;
 
-fn main() {
+#[async_std::main]
+async fn main() -> Result<(), Box<dyn Error>> {
```

Comment:
This doesn't need to have a result return? You aren't handling any errors in this function? Are you planning on doing this later?

Reply:
I'll remove it for now